sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -163,21 +189,23 @@ export class Application extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   pirDeniedReasonId?: PIRDeniedReason;
-
+  /**
+   * Custom reason why a Program Information Request (PIR) was denied.
+   */
```

Comment:
Please change to "Other reason why a Program Information Request (PIR) was denied."

Reply:
Changed!